nerdtree_plugin/fs_menu.vim

```diff
@@ -203,7 +203,7 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        let metadata = split(system('ls -ldh ' . shellescape(treenode.path.str())), '\n')
```

Comment:
I'm back. Sorry for the delay. What if we combine these ideas? It's ugly, but it could work. I don't have access to Linux, just OSX.

### In a bash shell...
```
x=$(stat -f "%z" vimrc) && y=$(echo $x | sed -e :a -e 's/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/;ta') && ls -ld vimrc | sed -e "s/ $x / $y /" && unset x && unset y
```

### In vimscript...

```
let metadata =
\   split(
\     system('x=$(stat -f "%z" ' .
\            shellescape(treenode.path.str()) .
\            ') && y=$(echo $x | sed -e :a -e "s/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/;ta") && ls -ld ' .
\            shellescape(treenode.path.str()) .
\            ' | sed -e "s/ $x / $y /" && unset x && unset y'),
\       '\n')
```